src/web/lib/decidePalette.ts

```diff
@@ -21,20 +24,58 @@ const textHeadline = (format: Format): string => {
 				case Design.Feature:
 					return pillarPalette[format.theme].dark;
 				case Design.Interview:
-					return neutral[100];
+					return WHITE;
```

Comment:
Should these be better as palette colours? I'm not sure with source these days!

Reply:
I think the purest definition of white from `source` is this `neutral[100]`. The rest seem very context oriented, like `background.primary` or `text.cta...`, neither of which is semantically true here